### PR TITLE
Fix initialism errors (PR-2) 

### PR DESCRIPTION
**Pull request by haardikdharma10**
#### Summary

For #16623 partially (This is PR-2)

#### Ticket Link

#16623  

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by agnivade — CHANGES_REQUESTED** [review]
Thanks @haardikdharma10 

We are moving the mlog and filestore packages into a separate directory. So there might be some merge conflicts to the other PR. Please revert the changes to those packages for now.

**Comment by haardikdharma10** [comment]
Done @agnivade! This is ready for review:)

**Comment by agnivade** [comment]
@haardikdharma10 - Thanks for doing the changes. Unfortunately, the PR has some conflicts now. Please resolve them when you get a chance. Thanks!

**Comment by haardikdharma10** [comment]
Conflicts resolved @agnivade!

File: plugin/context.go [hunk 1]
```diff
@@ -7,10 +7,10 @@ package plugin
 // For requests this is built in app/plugin_requests.go
 // For hooks, app.PluginContext() is called.
 type Context struct {
-	SessionId      string
-	RequestId      string
-	IpAddress      string
+	SessionID      string
+	RequestID      string
+	IPAddress      string
```
Comment (agnivade) [inline plugin/context.go:12]: @crspeller - Any concerns with changing these?

Comment (crspeller) [reply plugin/context.go:12]: Yes. It's a breaking change for plugins as they import this package and use these fields. Best to revert.

Comment (agnivade) [reply plugin/context.go:12]: Thanks. @haardikdharma10 - please revert these.

Comment (haardikdharma10) [reply plugin/context.go]: Done!

File: plugin/context.go [hunk 2]
```diff
@@ -12,5 +12,5 @@ type Context struct {
 	IpAddress      string
 	AcceptLanguage string
 	UserAgent      string
-	SourcePluginId string
+	SourcePluginID string
```
Comment (agnivade) [inline plugin/context.go:15]: This is still there.

**Review by agnivade — CHANGES_REQUESTED** [review]

**Review by agnivade — APPROVED** [review]

**Comment by haardikdharma10** [comment]
We have more conflicts @agnivade :( I'll try to resolve them.

**Review by isacikgoz — APPROVED** [review]

**Comment by haardikdharma10** [comment]
Same question here :)

cc @agnivade, @isacikgoz

**Comment by agnivade** [comment]
This is a bit weird. The translations are failing. But it passes locally. I'll need to check.

**Comment by haardikdharma10** [comment]
> This is a bit weird. The translations are failing. But it passes locally. I'll need to check.

This happened with a previous PR too. See https://github.com/mattermost/mattermost-server/pull/16877#issuecomment-781247538

**Comment by haardikdharma10** [comment]
Let's get this PR done @agnivade before we have more conflicts 😅

**Comment by agnivade** [comment]
There's something I might be missing here, because I can't get it to fail locally. Sorry, I am a bit swamped with work right now. @isacikgoz - if you get some time, would you mind taking a look at this?

**Comment by haardikdharma10** [comment]
@isacikgoz, gentle ping.

**Comment by agnivade** [comment]
Hey @haardikdharma10 - sorry for the delay here, can you please resolve the conflicts, and then we can look into why the build is failing.

**Comment by haardikdharma10** [comment]
> Hey @haardikdharma10 - sorry for the delay here, can you please resolve the conflicts, and then we can look into why the build is failing.

resolved:)

**Comment by agnivade** [comment]
Ok, it seems like it might have been something to do with the PR from a fork and not in an upstream branch. I also had to send a dummy PR to the EE repo.

There is definitely some mystery here as to how it got resolved. Chalking it up for some other time :)

**Pull request merged**